features/admin.validation.v1/constants/validation-config-constants.ts

```diff
@@ -41,6 +41,14 @@ export class ValidationConfigConstants {
         PASSWORD_MIN_VALUE: 5
     };
 
+    /**
+     * These scopes are checked to determine whether to display the new rule-based password expiry configuration UI.
+     * If these scopes are not available, legacy password expiry configuration will be shown for backward compatibility.
+     */
+    public static readonly RULE_BASED_PASSWORD_EXPIRY_REQUIRED_SCOPES: string[] = [
```

Comment:
Let's add a new feature object for this in the deployment config instead, so that deployment config would be the single source for maintaining scopes.